### ISSUE TITLE
Have this message appearing in a new repo, after composer command

### ISSUE THREAD

**Issue by SilberWitch** [nostrver-se/nostr-php#61]
![Screenshot of Terminal](https://i.nostr.build/YELpu90OFkw2moKW.png)

I went ahead and said, "Yes," but I didn't know if it was significant.

**Comment by Sebastix** [comment]
The latest tag / release is 1.4.0 and is not a dev release. It should be added in the dependencies section in your composer.json

https://packagist.org/packages/swentel/nostr-php

Maybe try to reinstall it?

'composer remove swentel/nostr-php'
'composer require swentel/nostr-php'

**Comment by SilberWitch** [comment]
Hmm. Just did that and it asked the same question, again. Check "no", this time.

**Comment by Sebastix** [comment]
Can I view the composer.json file?

**Comment by Sebastix** [comment]
Found it and learned something new here after reading https://php.watch/articles/composer-prompt-require-dev-dev-packages.

```
Composer inspects the keywords section of the composer.json file being installed, and if it contains dev, testing, or static analysis as keywords, Composer prompts the user to install it with --dev flag if it is not passed.
```

I've added some keywords in the `composer.json` (cc4df692d773e32425165221e7ce23779ddd5f26) and also `dev` there as a value, so that's why the package is handled as a dev dependency (which is not). I didn't know that.

I just fixed it in a new release [1.4.1](https://github.com/nostrver-se/nostr-php/releases/tag/1.4.1).
